crates/simd-doc/src/tests/fixtures.rs

```diff
@@ -60,11 +60,11 @@ fn test_simd_and_fallback_results_are_equal() {
     ]);
     let cases: Vec<Value> = serde_json::from_value(cases).unwrap();
 
-    // Build up an input fixture which has lots of whitespace, but consists of a whole documents.
+    // Build up an input fixture of whole documents.
     let mut input = Vec::new();
     for doc in cases.iter() {
-        serde_json::to_writer_pretty(&mut input, &doc).unwrap();
-        input.push(b'\t');
+        serde_json::to_writer(&mut input, &doc).unwrap();
+        input.push(b'\n');
```

Comment:
yes. The guarantee we provide for parse_many() or transcode_many() is that we handle newline-delimited JSON with no interior newlines. That's required because, without this restriction, we don't know where to resume from upon a JSON decoding error.

parse_one() continues to handle interior newlines.